src/content/learn/index.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Quick Start
+title: Mulai Cepat
 ---
 
 <Intro>
 
-Welcome to the React documentation! This page will give you an introduction to the 80% of React concepts that you will use on a daily basis.
+Selamat datang di dokumentasi React! Halaman ini akan memberikan Anda pengenalan tentang 80% konsep React yang akan Anda gunakan sehari-hari.
 
 </Intro>
 
 <YouWillLearn>
 
-- How to create and nest components
-- How to add markup and styles
-- How to display data
-- How to render conditions and lists
-- How to respond to events and update the screen
-- How to share data between components
+- Cara membuat dan menyarangkan (*nest*) komponen-komponen
+- Cara menambahkan *markup* dan penggayaan (*styles*)
```

Comment:
Need feedback for this:

- penggayaan (*styles*)
- gaya (*styles*)
- *styles*

cc @resir014 @r17x @mazipan @zainfathoni

Reply:
penggayaan (styles) ✅
gaya (styles)  ✅

Jika ragu dan mencari aman maka berikan istilah english-nya. 👍

Reply:
I think let's refer to other sites (e.g. MDN) and see what they refer to it as. If there's no clear reference let's keep it as `*styles*`. @mhaidarhanif @r17x

Reply:
@resir014 Okay how about to keep it as `*styles*` but without translation to `penggayaan` at all? Need to be confirmed with @r17x @zainfathoni @mazipan

Reply:
> Okay how about keeping it as `*styles* `but without translation to `penggayaan` at all?

Let's keep it as `*styles*`. I think "penggayaan" is too obscure of a translation to use. @mhaidarhanif

Reply:
Got it, I'll revert to *styles*

cc @zainfathoni @r17x @mazipan